index.php

```diff
@@ -10,29 +10,33 @@
     <title>Guitar Collection</title>
     <link rel="stylesheet" type="text/css" href="normalize-8-0-1.css">
     <link rel="stylesheet" type="text/css" href="styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Barlow|Cairo|Catamaran|Maven+Pro|Rajdhani&display=swap"
```

Comment:
You're only using Catamaran in the page styling - you can remove the unused ones.